packages/expo-av/src/__stories__/Audio.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import { Asset } from 'expo-asset';
+import * as React from 'react';
+import { PixelRatio, StyleSheet } from 'react-native';
+
+import AudioModeSelector from './components/AudioModeSelector';
+import AudioPlayer from './components/AudioPlayer';
+
+export const AudioHttpPlayer = () => (
+  <>
+    <AudioPlayer
+      source={{
+        uri:
+          // tslint:disable-next-line: max-line-length
+          'https://p.scdn.co/mp3-preview/f7a8ab9c5768009b65a30e9162555e8f21046f46?cid=162b7dc01f3a4a2ca32ed3cec83d1e02',
+      }}
+      style={styles.player}
+    />
+    <AudioModeSelector />
+  </>
+);
+
+AudioHttpPlayer.storyConfig = {
+  name: 'Play Remote Audio',
+};
+
+export const LocalAssetPlayer = () => (
+  <>
+    <AudioPlayer
+      source={Asset.fromModule(require('./assets/sounds/polonez.mp3'))}
+      style={styles.player}
+    />
+    <AudioModeSelector />
+  </>
+);
+
+LocalAssetPlayer.storyConfig = {
+  name: 'Play Local Audio',
+};
+
+export const ModeSelector = () => <AudioModeSelector />;
+
+ModeSelector.storyConfig = {
+  name: 'Audio Selector',
+};
```

Comment:
What about such construction where you wrap some components with additional required configuration passed as the second parameter? This way you hide any required static properties (`storyConfig` in this example) or any additional logic and focus just on the component code 🤔 
```suggestion
export const ModeSelector = useStory(() => <AudioModeSelector />, { name: 'Audio Selector' });
```

Reply:
I like this idea too, it would be a divergence from the CSF which is what I based a lot on: https://storybook.js.org/docs/react/api/csf 

but if it suites our needs better than I think its worth it. I'll think about this a bit more